pkg/server/util.go

```diff
@@ -81,22 +81,10 @@ func extractFamilyFromTCPConn(conn *net.TCPConn) int {
 	return family
 }
 
-func setsockOptString(sc syscall.RawConn, level int, opt int, str string) error {
-	var opterr error
-	fn := func(s uintptr) {
-		opterr = syscall.SetsockoptString(int(s), level, opt, str)
-	}
-	err := sc.Control(fn)
-	if opterr == nil {
-		return err
-	}
-	return opterr
-}
-
 func setsockOptInt(sc syscall.RawConn, level, name, value int) error {
 	var opterr error
 	fn := func(s uintptr) {
-		opterr = syscall.SetsockoptInt(int(s), level, name, value)
+		opterr = syscall.SetsockoptInt(syscall.Handle(s), level, name, value)
```

Comment:
would it be acceptable to do a switch/case block here, and use syscall.Handle() for Windows and int() for non-Windows?

That being said, I think the whole idea of Windows support could be a non-starter because Windows does not seem to support TCP_MD5SIG or IPPROTO_TCP with password hashing.  So this pull request may be stalled until Windows supports those.